packages/botonic-plugin-flow-builder/src/action.tsx

```diff
@@ -18,31 +19,33 @@ export class FlowBuilderAction extends React.Component<FlowBuilderActionProps> {
     const flowBuilderPlugin = getFlowBuilderPlugin(request.plugins)
     const locale = flowBuilderPlugin.getLocale(request.session)
     const payload = request.input.payload
-    let targetContentId: string | undefined = payload
+    let targetNode: HtNodeWithContent | string | undefined = payload
+
     if (!payload && request.session.is_first_interaction) {
-      targetContentId = await flowBuilderPlugin.getStartId()
+      targetNode = flowBuilderPlugin.cmsApi.getStartNode()
```

Comment:
better to create a constant with the name startTargetNode
`startTargetNode = flowBuilderPlugin.cmsApi.getStartNode()`

Reply:
Maybe it's a good idea, yes, but let's first try to agree about [these priorities](https://github.com/hubtype/botonic/pull/2531#discussion_r1234527629).